pootle/apps/pootle_project/views.py

```diff
@@ -71,7 +71,12 @@ def get_object(self):
                self.kwargs['filename']))
         regex = r"^/[^/]*%s$" % project_path
         if not self.kwargs["filename"]:
-            dirs = Directory.objects.live().select_related("parent__pootle_path")
+            dirs = Directory.objects.live()
+            if self.kwargs['dir_path'].count("/"):
```

Comment:
@unho `ProjectBrowseView` processes all x-language urls. So `/projects/firefox/browser/chrome/browser/safebrowsing/` is handled with `dir_path` and `/projects/firefox/browser/chrome/browser/customizableui/customizableWidgets.properties.po` is handled with `filename`.